R/human_infection.R

```diff
@@ -729,6 +749,15 @@ calculate_treated <- function(
     )
   }
   
+  # Update liver stage drug effects
+  if(length(parameters$drug_hynozoite_efficacy) > 0){
```

Comment:
Are both `if`s required here?
If `successfully_treated$successfully_treated_hypnozoites$size() > 0` wouldn't `length(parameters$drug_hynozoite_efficacy) > 0` by definition?

Reply:
The way I've coded it means that `successfully_treated$successfully_treated_hypnozoites` is only generated when `length(parameters$drug_hynozoite_efficacy) > 0`. So it doesn't exist for falciparum, or for vivax when non hypnozoite drugs are being used (e.g., just CQ).

Reply:
This might be avoidable if all vivax drugs have that parameter (see other comment)?